astropy/table/meta.py

```diff
@@ -165,7 +165,7 @@ def _get_col_attributes(col):
     attrs['name'] = col.info.name
 
     type_name = col.info.dtype.type.__name__
-    if six.PY3 and (type_name.startswith('bytes') or type_name.startswith('str')):
+    if six.PY3 and type_name.startswith(('bytes', 'str')):
```

Comment:
This will conflict with #5312. Should we wait till that one is merged first and then rebase this one?